ExotischNederland/DAL/SQLDAL.cs

```diff
@@ -19,6 +19,31 @@ public SQLDAL()
             this.connection = new SqlConnection(this.connectionString);
         }
 
+        public T Find<T>(string _field, string _value)
+        {
+            string table = typeof(T).Name;
+            var query = new QueryBuilder(table).Select().Where(_field, "=", _value).Build();
+            using (SqlConnection connection = new SqlConnection(this.connectionString))
+            {
+                connection.Open();
+                using (SqlCommand command = new SqlCommand(query, connection))
+                {
+                    using (SqlDataReader reader = command.ExecuteReader())
+                    {
+                        if (reader.Read())
+                        {
+                            // Use the fillable array to only fill the properties that are allowed
+                            var values = new Dictionary<string, object>();
+                            for (int i = 0; i < reader.FieldCount; i++)
```

Comment:
nitpick(non-blocking): maybe add a extra comment or 2 here, could be hard to understand for @paulib2000